src/managers/user.ts

```diff
@@ -236,6 +236,48 @@ export const sendDisable2faMail = async ({ user_id }: { user_id: number }) => {
   });
 };
 
+export const sendUpdatePersonalInformationEmail = async ({
+  user_id,
+  previousInformations,
+  newInformation,
+}: {
+  user_id: number;
+  previousInformations: Record<string, any>;
+  newInformation: Record<string, any>;
+}) => {
+  const user = await findById(user_id);
+  if (isEmpty(user)) {
+    throw new UserNotFoundError();
+  }
+  const { given_name, family_name, email } = user;
+
+  const getUpdatedFields = (
+    prev: Record<string, any>,
+    current: Record<string, any>,
+  ) => {
+    const allowedKeys = ["given_name", "family_name", "phone_number", "job"];
+    const changes: Record<string, { new: any; old: any }> = {};
```

Comment:
Can be string string ?
```suggestion
    const changes: Record<string, { new: string; old: string}> = {};
```